react-native/android/build.gradle

```
@@ -177,11 +178,35 @@ def findNdkBuildFullPath() {
         def ndkDir = System.getenv('ANDROID_NDK_HOME')
         return new File(ndkDir, getNdkBuildName()).getAbsolutePath()
     }
-    def ndkDir = android.hasProperty('plugin') ? android.plugin.ndkFolder :
-            plugins.getPlugin('com.android.library').sdkHandler.getNdkFolder()
-    if (ndkDir) {
-        return new File(ndkDir, getNdkBuildName()).getAbsolutePath()
+
+    def ndkDir = null
+
+    try {
+        // Reading [ndk.dir] from local.properties
+        Properties properties = new Properties()
+        properties.load(project.rootProject.file('local.properties').newDataInputStream())
+
+        ndkDir = properties.getProperty('ndk.dir')
+        if(ndkDir != null)
+            return new File(ndkDir, getNdkBuildName()).getAbsolutePath()
+    }catch(FileNotFoundException e){
```

```suggestion
    } catch(FileNotFoundException e) {
```
Just trying to keep the code-style consistent across the commit.